src/components/landing-page/CastDao.tsx

```
@@ -30,58 +35,73 @@ export default function CastDao() {
           in the US unless exempt from or not subject to registration under the
           Securities Act and complies with applicable US securities laws.
         </div>
-        <div className="flex flex-row items-center gap-6 text-2xl font-normal mt-[48px] max-md:mt-[16px] max-md:text-base">
-          <Link
+        <div className="flex flex-row items-center gap-6 text-2xl font-normal mt-[48px] max-md:mt-[16px] max-md:text-base max-md:justify-between max-md:gap-4">
+          <OpenLink2
+            frameSdk={sdk}
+            frameCtx={context}
             href={`${
               windowObj && windowObj.location.origin
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)